tests/test_geometric_line_utilities.py

```diff
@@ -0,0 +1,99 @@
+"""Test suite for the geometric_line_utilities module."""
+
+import numpy as np
+import pytest
+from stratigraphy.util.dataclasses import Line, Point
+from stratigraphy.util.geometric_line_utilities import (
+    _get_orthogonal_projection_to_line,
+    _merge_lines,
+    _odr_regression,
+)
+
+
+# Remember, phi is orthogonal to the line we are to parameterize
+# The way phi is defined is a bit counterintuitive, but it seems consistent all along.
```

Comment:
Yes, I'm also still confused by this. I would have expected the sign to be the other way around... Needs more investigation.

Reply:
Actually, now it makes more sense, after ensuring that r>0, as also recommended in the paper https://www.researchgate.net/publication/272179120_A_tutorial_on_the_total_least_squares_method_for_fitting_a_straight_line_and_a_plane